test/veOLAS.js

```diff
@@ -439,9 +439,9 @@ describe("Voting Escrow OLAS", function () {
             await ve.connect(owner).increaseAmount(oneOLABalance);
             await ve.connect(owner).increaseAmount(oneOLABalance);
 
-            // Get past votes of the owner
+            // Get past votes of the owner (bug resolved in wveOLAS)
             const votesOwner = await ve.getPastVotes(owner.address, blockNumber);
-            expect(Number(votesOwner)).to.greaterThan(0);
+            expect(votesOwner).to.greaterThan(0);
```

Comment:
Show that the original function is incorrectly returning non-zero voting power when asked for the block number before the first lock.